README.md

```diff
@@ -1,45 +1,51 @@
-[Meta analysis template](https://colab.research.google.com/drive/1NXYvi3eHOWlFHXzcg7Vhw3xNJpNXcqx1) | Use to analyse data from multiple publishers, or to perform other types of analysis on the Kingfisher database | :heavy_check_mark: | | | | |
-[Structure and format feedback template](https://colab.research.google.com/drive/1GmkA3kFL9k9MdTUln4pcRmc-KZneL5VB) | Use to provide feedback on structure and format issues reported by the Data Review Tool | :heavy_check_mark: | :heavy_check_mark: | :heavy_check_mark: | :heavy_check_mark: | :heavy_check_mark: |
-[Data quality feedback template](https://colab.research.google.com/drive/1Lj96xTde5GpFQ5hnvB2GYZ7gY4wuvUYt) | Use to provide detailed feedback on structure and format, conformance and coherence issues | :heavy_check_mark: | :heavy_check_mark: | :heavy_check_mark: | :heavy_check_mark: | :heavy_check_mark:
-Usability checks template | Use to provide feedback on data usability | | | | | |
+To open a component from a different branch, use Colab's [GitHub browser](https://colab.research.google.com/github/open-contracting/kingfisher_notebook_components/).
 
-## Contributing
+Alternatively, you can use the Open in Colab browser extension ([Chrome](https://chrome.google.com/webstore/detail/open-in-colab/), [Firefox](https://addons.mozilla.org/en-US/firefox/addon/open-in-colab/)) to add a button that, when clicked when viewing a Jupyter notebook on GitHub, will open that notebook in Colab.
+
+### Add a component
 
-### Add or edit a component
+1. [Create a new notebook](https://colab.research.google.com/#create=true)
+2. Set a title using H2 formatting and add your cells, following the [style guide for SQL statements](https://ocp-software-handbook.readthedocs.io/en/latest/python/code.html#sql-statements).
 
-1. [Create a new notebook](https://colab.research.google.com/#create=true), or open an existing component in Colab.
-2. Add or edit cells, following the [style guide for SQL statements](https://ocp-software-handbook.readthedocs.io/en/latest/python/code.html#sql-statements). For new components, set a title using H2 formatting.
-3. Format any SQL code you add or edit, using [pg_format](https://github.com/darold/pgFormatter) with the following options: `pg_format -f 1`. Alternatively, you can use the [online version](https://sqlformat.darold.net/) and set the **Functions** option to 'lowercase'.
+### Edit a component
+
+1. Open the component in Colab.
+2. Add or edit cells, following the [style guide for SQL statements](https://ocp-software-handbook.readthedocs.io/en/latest/python/code.html#sql-statements).
+
+### Format your code
+
+1. Format any SQL code you add or edit, using [pg_format](https://github.com/darold/pgFormatter) with the following options: `pg_format -f 1`. Alternatively, you can use the [online version](https://sqlformat.darold.net/) and set the **Functions** option to 'lowercase'.
```

Comment:
In practice, would an analyst need to do this for each cell? It seems faster to commit the notebook to the branch, then run pre-commit locally.

Reply:
If editing many cells, yes. I've removed this section in https://github.com/open-contracting/notebooks-ocds/pull/30/commits/c584451718fab92970860c6ce2354ce2e4726972 in favour of the lining message which prompts the users to run pre-commit locally.